bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/network/MagentaTVNetwork.java

```diff
@@ -0,0 +1,278 @@
+                                // Store it as a candidate to be returned if site-local address is not
+                                // subsequently found...
+                                // Note that we don't repeatedly assign non-loopback non-site-local addresses as
+                                // candidates, only the first. For subsequent iterations, candidateAddress will
+                                // be non-null.
+                                candidateAddress = inetAddr;
+                                condidateIFace = iface;
+                                logger.trace("local address candidate: {0}", candidateAddress.getHostAddress());
+                            }
+                        }
+                    }
+                }
+                if (localIP.isEmpty() && (candidateAddress != null)) {
+                    // We did not find a site-local address, but we found some other non-loopback
+                    // address.
+                    // Server might have a non-site-local address assigned to its NIC
+                    // (or it might be running IPv6 which deprecates the "site-local" concept)
+                    // So return this non-loopback candidate address...
+                    logger.trace("founbd local interface address: {0}", candidateAddress.getHostAddress());
+                    localIP = candidateAddress.getHostAddress();
+                    localInterface = condidateIFace;
+
+                }
+
+                if (localIP.isEmpty() || localIP.equals("0.0.0.0") || localIP.equals("127.0.0.1")) {
+                    logger.trace("getLocalHostAsString() is empty, try datagram socket");
+                    try (final DatagramSocket socket = new DatagramSocket()) {
+                        socket.connect(InetAddress.getByName("8.8.8.8"), 10002);
+                        localIP = socket.getLocalAddress().getHostAddress().toString();
+                    }
+                }
+                if (localIP.isEmpty() || localIP.equals("0.0.0.0") || localIP.equals("127.0.0.1")) {
+                    // At this point, we did not find a non-loopback address.
+                    // So fall back to returning whatever InetAddress.getLocalHost() returns...
+                    InetAddress jdkSuppliedAddress = InetAddress.getLocalHost();
+                    if (jdkSuppliedAddress == null) {
+                        throw new UnknownHostException(
+                                "The JDK InetAddress.getLocalHost() method unexpectedly returned null.");
+                    }
+                    localIP = jdkSuppliedAddress.getHostAddress();
+                    logger.trace("jdkSuppliedAddress: {0}", jdkSuppliedAddress.getHostAddress());
+                }
+            }
+
+            if (localIP.isEmpty() || localIP.equals("0.0.0.0") || localIP.equals("127.0.0.1")) {
+                {
+                    logger.fatal("Unable to detect local ip address!");
```

Comment:
change to logger .debug

Reply:
important info - this means: something is wrong in the network setup or the binding is unable to read it

Reply:
Add to Thing status info or logger.warn